### PR TITLE
Release/1.0.1

### PR DESCRIPTION
**Pull request by turegjorup**
Minor cleanup release. Currently the `1.0.0` tag doesn't show up on packagist because of errors in `composer.json`

```
Reading composer.json of danskernesdigitalebibliotek/ddb-cover-service-php-client (1.0.0)
Skipped tag 1.0.0, tag (1.0.0.0) does not match version (2.0.0.0) in composer.json
Reading composer.json of danskernesdigitalebibliotek/ddb-cover-service-php-client (1.0.0-rc1)
Skipped tag 1.0.0-rc1, tag (1.0.0.0-RC1) does not match version (2.0.0.0) in composer.json
Reading composer.json of danskernesdigitalebibliotek/ddb-cover-service-php-client (develop)
Importing branch develop (dev-develop)
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,24 +20,12 @@ PHP 5.5 and later
...
-    {
-      "type": "git",
-      "url": "https://github.com/danskernesdigitalebibliotek/ddb-cover-service-php-client.git"
-    }
-  ],
-  "require": {
-    "danskernesdigitalebibliotek/ddb-cover-service-php-client": "*@dev"
-  }
-}
+composer require composer require danskernesdigitalebibliotek/ddb-cover-service-php-client
```
Comment (cableman) [inline README.md]: That may be a compose require to much :-)

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+version: "3"
+
+services:
+  phpfpm:
+    image: itkdev/php8.0-fpm
+    environment:
+      - XDEBUG_MODE=${XDEBUG_MODE:-off}
+      - XDEBUG_SESSION=${XDEBUG_SESSION:-0}
+      - PHP_MAX_EXECUTION_TIME=30
+      - PHP_MEMORY_LIMIT=256M
```
Comment (cableman) [inline docker-compose.yml:11]: Why the extra memory and the time is the same as the default value

**Review by cableman — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (turegjorup) [reply README.md]: Fixed

File: docker-compose.yml (hunk 2 above)
Comment (turegjorup) [reply docker-compose.yml:11]: This is our default values for PHP containers?

Comment (turegjorup) [reply docker-compose.yml:11]: https://github.com/itk-dev/devops_itkdev-docker/blob/7da932868932b766079020171f071e34f758ac74/templates/symfony-6/docker-compose.yml#L29-L33

**Review by cableman — APPROVED** [review]

**Pull request merged**